<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 197, 255);</string>
  </property>
  <widget class="QLineEdit" name="C_senha">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_email">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmarSenha">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Confirmar Senha:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_ConfirmarSenha">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_usuario">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Usuario">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Nome Usuário:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="email">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;E-mail Usuário:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_login_T2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/USUARIO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Senha">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Senha:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Editar_nome">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Editar_email">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Editar_senha">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Excluir_usuario">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="aviso">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="voltar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <zorder>C_senha</zorder>
  <zorder>C_email</zorder>
  <zorder>ConfirmarSenha</zorder>
  <zorder>C_ConfirmarSenha</zorder>
  <zorder>C_usuario</zorder>
  <zorder>Usuario</zorder>
  <zorder>email</zorder>
  <zorder>Senha</zorder>
  <zorder>img_login_T2</zorder>
  <zorder>Editar_nome</zorder>
  <zorder>Editar_email</zorder>
  <zorder>Editar_senha</zorder>
  <zorder>Excluir_usuario</zorder>
  <zorder>aviso</zorder>
  <zorder>voltar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
